Cookbook-ios/Extensions/GeneralViewController.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  GeneralViewControllerExtension.swift
+//  Cookbook-ios
+//
+//  Created by Carolina Arcos on 1/8/20.
+//  Copyright © 2020 Wolox. All rights reserved.
+//
+
+import UIKit
+
+class GeneralViewController: UIViewController {
+    var feature: Features!
```

Comment:
I think this could be a `let`.
In `CarouselViewController` you could assign this property in the initializer.